utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt

```diff
@@ -92,6 +102,7 @@ class DataProvidersTest {
   @Before
   fun setUp() {
     setUpTestApplicationComponent()
+    MockitoAnnotations.initMocks(this)
```

Comment:
I don't think we need this anymore since we are manually mocking the class below (i.e. doing `Mockito.mock(CrashlyticsWrapper::class` below)

Reply:
Done